src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26864.cs

```diff
@@ -0,0 +1,26 @@
+#if MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26864 : _IssuesUITest
+	{
+		public Issue26864(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Shell Content Title Not Rendering in Full-Screen Mode on Mac Catalyst";
+
+		[Test]
+		[Category(UITestCategories.Shell)]
+		public void ShellContentTitleNotRendering()
+		{
+			App.WaitForElement("Settings");
+			App.EnterFullScreen();
```

Comment:
We are removing some system animations running the UITests on macOS, but still, the change to full screen is not something immediate. Here, we need to wait a little bit.

Reply:
> We are removing some system animations running the UITests on macOS, but still, the change to full screen is not something immediate. Here, we need to wait a little bit.

I have added a 500-millisecond delay after switching to full screen. Please let me know if you have any concerns regarding this change?